witness/golang/cmd/witness/internal/witness/witness.go

```diff
@@ -126,8 +127,16 @@ func (w *Witness) GetCheckpoint(logID string) ([]byte, error) {
 }
 
 // Update updates the latest checkpoint if nextRaw is consistent with the current
-// latest one for this log. It returns the latest cosigned checkpoint held by
-// the witness, which is a signed version of nextRaw if the update was applied.
+// latest one for this log.
+//
+// It returns the latest cosigned checkpoint held by the witness, which is a signed
+// version of nextRaw if the update was applied.
+//
+// If an error occurs, this method will generally return an error with a status code:
+// - codes.NodeFound if the log is unknown
```

Comment:
```suggestion
// - codes.NotFound if the log is unknown
```

Reply:
_*cough*_ Signs of one too many Merkle implementations... :)